### PR TITLE
add sortedTokenList selector

### PR DESCRIPTION
**Pull request by Velenir**
Sorting tokenList by ETH first, then balance DESC, then symbol

*pending order clarification*

### CODE REVIEW

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr406--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr406--dxreact.review.gnosisdev.com

File: src/components/TokenOverlay/index.tsx [hunk 1]
```diff
@@ -73,24 +73,56 @@ const prefilterByAvailableAuctions = createSelector(
   },
 )
 
+const sortedTokeList = createSelector(
```
Comment (W3stside) [inline src/components/TokenOverlay/index.tsx]: Toke(n)?

Comment (Velenir) [reply src/components/TokenOverlay/index.tsx]: But of course

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr406--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr406--dxreact.review.gnosisdev.com

**Review by W3stside — APPROVED** [review]

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr406--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr406--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr406--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr406--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr406--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr406--dxreact.review.gnosisdev.com

**Pull request merged**